src/components/ADempiere/Form/ProductInfo/productList.vue

```diff
@@ -83,15 +120,24 @@
         width="100"
       >
         <template slot-scope="scope">
-          {{ formatQuantity(scope.row.quantityAvailable) }}
+          {{
```

Comment:
Just can see change for soiurce code with the same functionality, please let it without changes